MapboxVision/VisionManager/VisionReplayManager.swift

```diff
@@ -43,14 +43,27 @@ public final class VisionReplayManager: BaseVisionManager {
      - Returns: Instance of `VisionReplayManager` configured to use data from specified session.
      */
     public static func create(recordPath: String) throws -> VisionReplayManager {
-        return VisionReplayManager(dependencies: try ReplayDependencies.default(recordPath: recordPath))
+        VisionReplayManager(dependencies: try ReplayDependencies.default(recordPath: recordPath))
     }
 
     /**
      Video source that provides frames from recorded video.
      */
     public var videoSource: VideoSource {
-        return dependencies.player
+        dependencies.player
+    }
+
+    public var duration: Float {
```

Comment:
Please add documentation for the new API.

Reply:
done